pandas/core/indexes/range.py

```diff
@@ -1165,6 +1165,43 @@ def any(self, *args, **kwargs) -> bool:
 
     # --------------------------------------------------------------------
 
+    # error: Return type "RangeIndex | Index" of "round" incompatible with
+    # return type "RangeIndex" in supertype "Index"
+    def round(self, decimals: int = 0) -> Self | Index:  # type: ignore[override]
+        """
+        Round each value in the Index to the given number of decimals.
+
+        Parameters
+        ----------
+        decimals : int, optional
+            Number of decimal places to round to. If decimals is negative,
+            it specifies the number of positions to the left of the decimal point.
```

Comment:
I know this is from the original docstring, but for me it's not obvious what this means by reading it, maybe adding an example or two here would be helpful.

Reply:
Sure thing. I added an example for the negative case.